R/wwr_report.R

```diff
@@ -85,7 +85,7 @@ wwr_export <- function(data, year, month, mean_station_pressure, mean_sea_level_
   if (!missing(mean_station_pressure)) {
     df_2_means <- data %>%
       dplyr::group_by(!!! rlang::syms(c(link, year))) %>%
-      dplyr::summarise(mean = sprintf("%6s", round(summary_mean(.data[[mean_station_pressure]], na.rm = TRUE), 1)), .groups = "keep")
+      dplyr::summarise(mean = sprintf("%6s", round(epitab::summary_mean(.data[[mean_station_pressure]], na.rm = TRUE), 1)), .groups = "keep")
```

Comment:
Interesting! I think that `summary_mean` is actually a function defined elsewhere in R-Instat in a different R file. We might for now want to not export this one until that one is sorted. I'll write an issue on this?